src/app/tests/suites/certification/Test_TC_OO_2_3.yaml

```diff
@@ -894,8 +898,8 @@ tests:
       PICS: OO.S.A4002 && PICS_SKIP_SAMPLE_APP && OO.S.C42.Rsp
       response:
           constraints:
-              minValue: 170
-              maxValue: 230
+              minValue: 200 - PIXIT.OO.MaxCommunicationTurnaround
```

Comment:
In 20a the `OffWaitTime` countdown starts.
In first 20b `readAttribute` of `OnOff` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In second 20b `readAttribute` of `OnTime` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In third 20b `readAttribute` of `OffWaitTime` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In 21a the `OnWithTimedOff` command is called, that should not affect countdown, but it still can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In first 21b `readAttribute` of `OnOff` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In second 21b `readAttribute` of `OnTime` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In third 21b `readAttribute` of `OffWaitTime` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
Additionally I think that 15% tolerance applies here, it elapsed 10 seconds since the countdown starts, 15% of 100 is 15, so in total `185`.
So in total I think it should be `185 - 7 * PIXIT.OO.MaxCommunicationTurnaround`.

Reply:
Updated